### PR TITLE
SCMOD-6825: Jep upgraded to 3.8.2 for license issues

### PR DESCRIPTION
**Pull request by thilagasanthosh**
https://portal.digitalsafe.net/browse/SCMOD-6825

http://sou-jenkins2.hpeswlab.net/job/CAFDataProcessing/view/Developer/job/CAFDataProcessing~worker-markup~SCMOD-6825~CI/

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -119,7 +119,7 @@
             <dependency>
                 <groupId>com.github.cafdataprocessing</groupId>
                 <artifactId>util-email-content-segregation</artifactId>
-                <version>2.0.0-5</version>
+                <version>2.1.0-SCMOD-6825-SNAPSHOT</version>
```
Comment (dermot-hardy) [inline pom.xml]: Don't forget to remove branch before merging.

File: worker-markup-container-fs/pom.xml [hunk 2]
```diff
@@ -297,7 +297,7 @@
                             <alias>worker-markup-fs</alias>
                             <name>${dockerDataProcessingOrg}worker-markup${dockerProjectVersion}</name>
                             <build>
-                                <from>cafapi/opensuse-jeptalon:1</from>
+                                <from>cafinternal/prereleases:opensuse-jeptalon-1.2.0-SCMOD-6825-SNAPSHOT</from>
```
Comment (dermot-hardy) [inline worker-markup-container-fs/pom.xml]: Don't forget to remove branch before merging.

**Review by dermot-hardy — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (thilagasanthosh) [reply pom.xml]: Removed branch build version

File: worker-markup-container-fs/pom.xml (hunk 2 above)
Comment (thilagasanthosh) [reply worker-markup-container-fs/pom.xml]: Removed branch build version

**Pull request merged**